packages/core/src/Dropdown/stories/Dropdown.stories.tsx

```diff
@@ -30,6 +37,16 @@ const widthDecorator: Decorator = (Story) => (
 export default {
   title: "Components/Dropdown",
   component: HvDropdown,
+  // @ts-expect-error https://github.com/storybookjs/storybook/issues/20782
+  subcomponents: { HvBaseDropdown },
+  parameters: {
```

Comment:
Can we remove this? It's not needed. We already have the `play` functions on the stories that need interactions.